coalib/output/ConsoleInteraction.py

```diff
@@ -164,35 +158,48 @@ def print_lines(console_printer,
     :param sourcerange:     The SourceRange object referring to the related
                             lines to print.
     """
+
     for i in range(sourcerange.start.line, sourcerange.end.line + 1):
         console_printer.print(format_lines(lines='', line_nr=i),
                               color=FILE_LINES_COLOR,
                               end='')
```

Comment:
hey this is for printing the line number

![there](https://cloud.githubusercontent.com/assets/5689132/17547762/74bd9b12-5f06-11e6-8337-428360f6fa4f.png)

And if you remove that line:

![missing](https://cloud.githubusercontent.com/assets/5689132/17547780/7f4c65cc-5f06-11e6-837c-0cf88ba0bf41.png)